src/sentry/rules/actions/notify_event_service.py

```diff
@@ -50,21 +53,40 @@ def after(self, event, state):
             self.logger.info('rules.fail.is_configured', extra=extra)
             return
 
-        plugin = plugins.get(service)
-        if not plugin.is_enabled(self.project):
-            extra['project_id'] = self.project.id
-            self.logger.info('rules.fail.is_enabled', extra=extra)
-            return
-
-        group = event.group
-
-        if not plugin.should_notify(group=group, event=event):
-            extra['group_id'] = group.id
-            self.logger.info('rule.fail.should_notify', extra=extra)
-            return
-
-        metrics.incr('notifications.sent', instance=plugin.slug)
-        yield self.future(plugin.rule_notify)
+        app = None
+        try:
+            app = SentryApp.objects.get(slug=service)
+        except SentryApp.DoesNotExist:
+            pass
+
+        if app:
+            kwargs = {'sentry_app': app}
+            metrics.incr('notifications.sent', instance=app.slug)
+            yield self.future(notify_sentry_app, **kwargs)
+        else:
+            plugin = plugins.get(service)
+            if not plugin.is_enabled(self.project):
+                extra['project_id'] = self.project.id
+                self.logger.info('rules.fail.is_enabled', extra=extra)
+                return
+
+            group = event.group
+
+            if not plugin.should_notify(group=group, event=event):
+                extra['group_id'] = group.id
+                self.logger.info('rule.fail.should_notify', extra=extra)
+                return
+
+            metrics.incr('notifications.sent', instance=plugin.slug)
+            yield self.future(plugin.rule_notify)
+
+    def get_installs(self):
```

Comment:
Maybe we call this something like `get_sentry_app_services`? "installs" is pretty vague, imo.